.changeset/few-sloths-raise.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-catalog': minor
+'@backstage/plugin-catalog-react': minor
```

Comment:
Since this is a non-breaking change and we're on `0.x` we can make these `patch`